README.md

```diff
@@ -122,19 +128,31 @@ if you need additional guidance with this task.
     for suggestions on how to interpret and apply the Responsible AI dashboard.  
     <br />
 
-1. **Health Care**
+2. **Health Care**
     In addition to several components that can help you improve model performance, 
     this scenario highlights a causal analysis tool that can guide real-world action. 
     This notebook takes about **25-30 minutes** to create the dashboard.  
     
-    - Follow this link to the [Jupyter notebook](./healthcare_story/Covid_Healthcare_Dashboard.ipynb). 
+    - Follow this link to the [Jupyter notebook](./education_story/Education_Dashboard.ipynb). 
         - If you choose not to clone this repo to Azure ML studio, 
         check [this link](./documentation/uploadjupyternotebookandrun.md) for guidance on uploading the notebook and data files.  
     - Once you have finished running the notebook,
-    refer to this [Health Care scenario guide](./documentation/healthCareExample.md) 
+    refer to this [Health Care scenario guide](./documentation/educationExample.md) 
     for suggestions on how to interpret and apply the Responsible AI dashboard.  
     <br />
 
+3. **Education**
+    In addition to several components that can help you improve model performance, 
+    this scenario highlights a causal analysis tool that can guide real-world action. 
+    This notebook takes about **25-30 minutes** to create the dashboard.  
+    
+    - Follow this link to the [Jupyter notebook](./healthcare_story/Covid_Healthcare_Dashboard.ipynb). 
```

Comment:
it looks like you mixed up the links to covid and education here, can you please fix here and above?